src/core/server/http/logging/get_response_log.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import querystring from 'querystring';
+import { isBoom } from '@hapi/boom';
+import type { Request } from '@hapi/hapi';
+import numeral from '@elastic/numeral';
+import { LogMeta } from '@kbn/logging';
+import { EcsEvent, Logger } from '../../logging';
+import { getResponsePayloadBytes } from './get_payload_size';
+
+const ECS_VERSION = '1.7.0';
+const FORBIDDEN_HEADERS = ['authorization', 'cookie', 'set-cookie'];
+const REDACTED_HEADER_TEXT = '[REDACTED]';
+
+// We are excluding sensitive headers by default, until we have a log filtering mechanism.
+function redactSensitiveHeaders(headers: Record<string, unknown>): Record<string, unknown> {
+  Object.keys(headers).forEach((key) => {
+    if (FORBIDDEN_HEADERS.includes(key.toLowerCase())) {
+      headers[key] = REDACTED_HEADER_TEXT;
```

Comment:
Thanks for catching this, will update to create a shallow copy.

Also, didn't use `omit` here because we had decided to censor the headers instead of simply stripping them out.

Reply:
> Also, didn't use omit here because we had decided to censor the headers instead of simply stripping them out.

Fair enough